content/22.vaccines.md

```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
 These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
 Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].
 
+The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms.
```

Comment:
Consider adding citation here too or rearranging the paragraph break?

Reply:
```suggestion
The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
```